### DISCUSSION TITLE
How to use environment variable/strore a data in the variable from the response. 'A' request has the response having token(object), I want this token value in the 'B' request. How to fetch and pass the value?

### DISCUSSION THREAD

**Discussion by Debolinac** [postmanlabs/postman-flows#167] [Q&A]
How to use environment variable/strore a data in the variable from the response. 'A' request has the response having token(object), I want this token value in the 'B' request. How to fetch and pass the value?

**Comment by saswatds** [comment]
Hey @Debolinac you can refer to this blog https://beththetester.com/2021/12/13/postman-flows-5-example-flows/ that one of our community members has created.

**Reply by Debolinac** [reply]
@saswatds Video is not useful for me.Kindly help
